backend/src/Entity/AdminProfileEntity.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AdminProfileEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+#[ORM\Entity(repositoryClass: AdminProfileEntityRepository::class)]
+class AdminProfileEntity
```

Comment:
the image field is missing.
- please do it with relation to the image entity.
- adminUserId it is much better if we call it just user.
- and it should be relation with the user entity.